source/index.html.md

```diff
@@ -630,6 +770,35 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects.
+Currently only the JWE is accepted.
+
+<dl class="dl-vertical">
+  <dt>
+    samsungpay[jwt]
+    <span class="type">
+      <a href="https://tools.ietf.org/html/rfc7516#section-3.1">
+        Compact serialized JWE token
+      </a>
+    </span>
+  </dt>
+  <dd>
+    Complete JWE token as received from Samsung Pay.
```

Comment:
Perhaps include an example snippet? (Cf. https://github.com/clearhaus/gateway-api-docs/pull/67)

Reply:
W3 isn't too happy about the example though; see 2nd error of https://validator.w3.org/nu/?doc=https%3A%2F%2Fdocs.gateway.clearhaus.com%2F 🙂

Reply:
Good idea to add examples :+1: 

I was planning to, but honestly I was considering finding some JS/CSS-thing that could help us with collapsible examples.